src/config/dynamic-theme-fixes.config

```
@@ -45,6 +45,15 @@ div[class^='scrollableList'] {
 
 ================================
 
+*.paramountnetwork.com
```

```suggestion
paramountnetwork.com
```
Re-order as appropriate.